pkg/cache/cache_ristretto.go

```diff
@@ -67,3 +68,8 @@ var _ Cache = (*wrapped)(nil)
 
 func (w wrapped) GetMetrics() Metrics                   { return w.Cache.Metrics }
 func (w wrapped) MarshalZerologObject(e *zerolog.Event) { e.EmbedObject(w.config) }
+
+func (w wrapped) Close() {
```

Comment:
Don't we need to add the Close method to the Cache interface and call it in dispatcher?

Reply:
`wrapped` already implements ristretto `Cache` interface, which also has the `Close` method.
Ristretto Cache was already being closed through the `CachingDispatcher`

Reply:
oh i totally missed that there was already a close lmao